### DISCUSSION TITLE
Get relation display fields

### DISCUSSION THREAD

**Discussion by Oggy4th** [pocketbase/pocketbase#2658] [Q&A] [answered]
Is it possible to get relation display fields without expand?
![image](https://github.com/pocketbase/pocketbase/assets/44497354/b4161afc-f8fd-49c3-a827-eb6cad95d649)

**Comment by ganigeorgiev** [comment, marked as answer]
I'm not entirely sure that I understand what do you mean, but you have to use `expand` to return the relation fields in the API response (_or send a separate request to the related collection_).

If you want to provide access to only a subset of the collection fields, you can create a View collection with a query like:

```sql
SELECT
    id,
    lastName,
    firstName,
    someOtherField
FROM employees
```

**Reply by Oggy4th** [reply]
Ohh I'll create new `SafeUser` View Collection and `getOne` by ID so the sensitive data doesn't go through
